<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastro_novo_password</class>
 <widget class="QWidget" name="cadastro_novo_password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager Password</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/Clowns_Password.ico</normaloff>../img/Clowns_Password.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="user_voltar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/seta.png</normaloff>../img/seta.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="service_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="service_user">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="service_password">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Service Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>service user</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>service password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="salvar_dados">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>not be shared with anyone</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>the saved data will </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/security.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="notifica">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
